Testing/VelaptorTests/NativeInterop/OpenGL/OpenGLServiceTests.cs

```diff
@@ -384,12 +386,18 @@ public void LabelVertexArray_WhenInvoked_LabelsVertexArray(string label, string
     public void LabelBuffer_WithInvalidBufferType_ThrowsException()
     {
         // Arrange
-        var exceptionMsg = "Exception of type 'System.ArgumentOutOfRangeException' was thrown. ";
-        exceptionMsg += $"(Parameter 'bufferType'){Environment.NewLine}Actual value was 123.";
+        var invalidValue = 123;
```

Comment:
Please convert this to a `const`.

```suggestion
        const invalidValue = 123;
```